Backend/Backend/Infrastructure/FileStorageHelper.cs

```diff
@@ -50,6 +50,8 @@ public static class FileStorageHelper
     /// Identifies the file type for transcript files.
     /// </summary>
     public const string TRANSCRIPT_FILE_TYPE = ".json";
+    
+    public static readonly string[] AVATAR_EXTENSIONS = new[] { ".jpg", ".png", ".jpeg", ".gif"};
```

Comment:
Are you sure we should allow .gif lol

Reply:
Could be fun I guess having animated avatars xD

Reply:
true lol